modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/scanner/InboxScannerService.java

```diff
@@ -289,8 +304,18 @@ public static String getCfg(Dictionary d, String key) throws ConfigurationExcept
     return ps;
   }
 
+  public static String getCfgOrDefault(Dictionary d, String key, String defaultValue) throws ConfigurationException {
```

Comment:
This is basically a custom re-implementation of Java's built-in [`Objects.toString(Object o, String nullDefault)`](https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#toString(java.lang.Object,%20java.lang.String)) with the only difference that

```java
getCfgOrDefault(properties, WORKFLOW_DEFINITION, null);
```
would be
```java
Objects.toString(properties.get(WORKFLOW_DEFINITION), null);
```
which isn't much of a difference.

Reply:
I debated this a lot and came to the conclusion that I wont touch this

Reply:
While I agree that our codebase perhaps has a little bloat (c.f OsgiUtil.getOptCfg, which does basically the same thing here without the default), why add a bunch of boilerplate to this relatively simple built-in?  I'm fine adding boilerplate, but I feel like in this case the extra code doesn't actually add anything.